core/Utils.vala

```diff
@@ -76,11 +76,49 @@ namespace Maya.Util {
     }
 
     /**
-     * Converts the given TimeType to a DateTime.
+     * Gets the timezone of the given TimeType as a GLib.TimeZone.
      */
     private TimeZone timezone_from_ical (ICal.Time date) {
```

Comment:
To avoid confusion with ICal.TimeZone better to explicitly state GLib.TimeZone here.

Reply:
Good point. I just pushed a new commit to make this explicit.